internal/runners/clean/run_lin_mac.go

```diff
@@ -34,7 +34,9 @@ func (u *Uninstall) runUninstall() error {
 	}
 
 	err = removeInstall(u.cfg)
-	if err != nil {
+	if errors.Is(err, errDirNotEmpty) {
+		u.out.Notice(locale.Tl("uninstall_warn_not_empty", "[WARNING]Unable to remove all files in the directory {{.V0}}[/RESET]"))
```

Comment:
Can't we give a reason here? It sounds like this would happen if the install dir had files that we don't recognize.